x-pack/plugins/infra/public/pages/metrics/hosts/components/tabs/alerts/alerts_tab_content.tsx

```diff
@@ -101,14 +101,14 @@ const MemoAlertSummaryWidget = React.memo(
     const { charts, triggersActionsUi } = services;
     const { getAlertSummaryWidget: AlertSummaryWidget } = triggersActionsUi;
 
-    const chartThemes = {
+    const chartProps = {
```

Comment:
For `onBrushEnd`, it is optional, and the logic will be provided by the consumer.

For `theme` and `baseTheme`, since `AlertSummaryWidget` is a shared component and these props are coming from the charts plugin, we decided to pass them as props to make the dependency on the charts plugin visible.